### DISCUSSION TITLE
InvalidOperationException: Queue empty

### DISCUSSION THREAD

**Discussion by muigokulol** [ORelio/Minecraft-Console-Client#2104] [Q&A]
I was trying to connect to a server (play.minecadia.com (1.8.9)) and got this error... Default settings are used. Does anyone know how to fix this?

![error](https://user-images.githubusercontent.com/83993189/184817018-a501b67a-2920-4934-96f0-f27f72507549.png)

**Comment by milutinke** [comment]
Have you tried connecting to other servers?
Which version of the client you're running?
